ui/page/main_page.go

```diff
@@ -258,20 +247,28 @@ func (mp *MainPage) UnlockWalletForSyncing(wal *dcrlibwallet.Wallet) {
 
 			return false
 		}).Show()
-
 }
 
 func (mp *MainPage) Handle() {
-	for mp.minimizeNavDrawerButton.Button.Clicked() {
-		mp.isNavDrawerMinimized = true
+	mp.drawerNav.CurrentPage = mp.currentPageID()
+	mp.appBarNav.CurrentPage = mp.currentPageID()
+
+	for mp.drawerNav.MinimizeNavDrawerButton.Button.Clicked() {
+		mp.drawerNav.Axis = layout.Vertical
```

Comment:
Add a toggle function in components/nav_drawer.go to switch between maximized and minimized attributes.